sentry_sdk/integrations/__init__.py

```diff
@@ -139,7 +138,7 @@ def setup_integrations(
                             "deprecated. Use `setup_once`.",
                             identifier,
                         )
-                        integration.install()
+                        integration.install()  # type: ignore
```

Comment:
This will anyway be removed in scope of https://github.com/getsentry/sentry-python/issues/2583

Reply:
This whole part: https://github.com/getsentry/sentry-python/blob/2f05ccbae7298978d2f1a8774a07386a018bcce9/sentry_sdk/integrations/__init__.py#L136-L142
Basically, setting up integration with `install` has been deprecated in favor of `setup_once` for some time now, so we can drop this fallback.